### ISSUE TITLE
Issue building with version 1.1.1 due to incomplete POM

### ISSUE THREAD

**Issue by ibevilinc** [signalfx/splunk-otel-android#634]
With gradle 8.3
   > Could not resolve com.splunk:splunk-otel-android:1.1.1.
     Required by:
         project :XXX
      > Could not resolve com.splunk:splunk-otel-android:1.1.1.
         > Could not parse POM https://repo1.maven.org/maven2/com/splunk/splunk-otel-android/1.1.1/splunk-otel-android-1.1.1.pom
            > Required version must not be null

Comparing the POM with earlier versions it seems that the issue is that the BOM version for io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha is missing. Adding a <version> entry resolves the issue.

```
  <dependencyManagement>
    <dependencies>
...
      <dependency>
        <groupId>io.opentelemetry</groupId>
        <artifactId>opentelemetry-bom-alpha</artifactId>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
    </dependencies>
  </dependencyManagement>
```

**Comment by breedx-splk** [comment]
Hi @ibevilinc. Thanks for bringing this up.

Curious if you added a dependency on `com.splunk:opentelemetry-android-instrumentation:1.1.1` in your project as well? There is a new dependency, but this may be unrelated to the problem you're reporting.

In any case, it's my expectation that the otel instrumentation bom should have enough information to resolve the otel bom. Maybe that's not working as expected...we will have to take a look.

**Comment by breedx-splk** [comment]
This was fixed in #627 and the dependency on the opentelemetyr-bom has been removed in favor of the otel instrumentation bom.

**Comment by ibevilinc** [comment]
I only changed the version from `1.0.0` to `1.1.1` and when I tried to run the build got that error. I just tried adding that dependency and that lets gradle get past the error...however I had to use `com.splunk:opentelemetry-android-instrumentation:1.1.1-alpha` for it to work since there's no plain `1.1.1` version.
<img width="785" alt="Screenshot 2023-09-06 at 5 51 09 PM" src="https://github.com/signalfx/splunk-otel-android/assets/1529805/d2ca151d-7a83-4db0-8c3f-8a9c1b11110e">

**Comment by ibevilinc** [comment]
Thought adding the additional dependency made a difference. However, the error popped up again when doing another build with `--refresh-dependencies` as part of the build parameters.

**Comment by breedx-splk** [comment]
Good call out on `1.1.1-alpha` vs. `1.1.1` for the `opentelemetry-android-instrumentation` dependency. I believe that both the coordinates and the version will change in the next release, as we incorporate the upstream opentelemetry artifacts.

**Comment by breedx-splk** [comment]
I think this should be resolved after the 1.2.0 release. Please reopen if there are still problems.
